api/channels_api_test.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import division
+from __future__ import print_function
+
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import testing_config  # Must be imported first
+import mock
+import unittest
+
+from api import channels_api
+
+
+class ChannelsAPITest(unittest.TestCase):
+
+  @mock.patch('api.channels_api.fetch_chrome_release_info')
+  @mock.patch('api.channels_api.fetchchannels.get_omaha_data')
```

Comment:
I have patched objects in the namespace where they are used.

Reply:
Hmm, I would expect it to be
@mock.patch('internals.fetchchannels.get_omaha_data')
If you see tests passing, then I suppose it works either way, but I think specifying the location where a function is defined is more conventional.

Reply:
Yes the test was a success. Anyway, I have changed it to what you suggested thanks.